runtime/syntax/dockerfile.vim

```diff
@@ -15,7 +15,7 @@ syntax case ignore
 
 syntax match dockerfileKeyword /\v^\s*(ONBUILD\s+)?(ADD|ARG|CMD|COPY|ENTRYPOINT|ENV|EXPOSE|FROM|HEALTHCHECK|LABEL|MAINTAINER|RUN|SHELL|STOPSIGNAL|USER|VOLUME|WORKDIR)\s/
 
-syntax match dockerfileKeyword /\v(AS)/
+syntax match dockerfileKeyword /\(^\s*FROM\s\+[^\s]\+\s\+\)\@<=AS\ze /
```

Comment:
`[^\s]` means `neither '\' nor 's'`, so should use `\S`.

Reply:
Fixed—thank you. I see that this is even [described](/vim/vim/blob/7a641ca1e113c813fc39cca26f69cd6fc19f71c2/runtime/doc/pattern.txt#L1179) in the manual.